test/isochrone.cc

```diff
@@ -30,7 +30,7 @@ using rp = rapidjson::Pointer;
 
 namespace {
 
-const auto config = test::make_config("test/data/utrecht_tiles");
+const auto cfg = test::make_config("test/data/utrecht_tiles");
```

Comment:
yeah this is an inconvenience. not being able to name something `config` when inside the valhalla namespace, the good news is that in a future pr we wont need to even keep the variable around we can just always call `config()` whereever we are in the code (in most scenarios)